test/parallel/test-debugger-repeat-last.js

```diff
@@ -10,7 +10,7 @@ const fixture = path.join(
 
 const args = [
   'debug',
-  `--port=${common.PORT}`,
+  `--port=0`,
```

Comment:
Sorry, ESLint: `Strings must use singlequote` Could you replace backticks with singlequotes, please.